foundations/installations/command_line_basics.md

```diff
@@ -60,6 +60,11 @@ But once you've typed in a little bit more, it will complete the name for you, m
 
 Third, there's a really handy shortcut for opening everything within a project directory: `.` Once you've installed a text editor, you can use this shortcut to open up an entire project and all of its files in one go. This shortcut is also commonly used with Git (which is covered in detail later on) with commands like `git add .` to add all of the files inside of a directory into Git's staging area. For example, if you have VS Code installed, you can `cd` into the project directory and then type `code .` (with the period) to open up all of the project files. See the next section of this lesson for a more detailed example.
 
+**A Note on typing passwords**:
+
+When using a command in the terminal that requires you to enter your password for authentication (such as
+`sudo`), the characters should not be visible to you as you type them. Don't worry, this is a security feature to protect the confidential info and your password is still being typed in the terminal. Just press `ENTER` when you are done.
+
```

Comment:
Since we're using an inline heading here, there shouldn't be a line break between lines 63 and 65. There also seems to be an unneccessary line break on line 65. This should instead look like (plus slight verbiage tweak):

```suggestion
**A Note on typing passwords**: When using a command in the terminal that requires you to enter your password for authentication (such as `sudo`), the characters should not be visible to you as you type them. While you might think the terminal isn't responding, don't worry! This is a security feature to protect confidential information, similar to how password fields on websites use asterisks or dots. Your password is still being typed in the terminal; just press `ENTER` when you are done.
```